### PR TITLE
Extract a lot more I18n strings

### PR DESCRIPTION
**Pull request by olivierlacan**
I've noticed while working on the French version that we were missing
a lot of interface strings, so I decided to look for obvious one to add
in order to make the newly available translated interfaces more
comprehensive.

I don't believe this is comprehensive but it improves the experience with 
a different locale quite a bit. It should also lower the translation percentages 
for locales other than English here but we should be able to improve this thanks 
to: https://www.localeapp.com/projects/8883

/cc @dwradcliffe @arthurnn

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -125,12 +126,22 @@ fr:
         wiki: "Wiki"
         mail: "Liste de diffusion"
         bugs: "Suivi de bugs"
+        download: "Télécharger"
+        subscribe: "Abonnement"
```
Comment (arthurnn) [inline config/locales/fr.yml]: should this be an verb instead? `s'abonner`

File: test/unit/helpers/rubygems_helper_test.rb [hunk 2]
```diff
@@ -60,6 +60,7 @@ class RubygemsHelperTest < ActionView::TestCase
     version = build(:version)
     linkset = build(:linkset, docs: nil)
 
+    @virtual_path = "rubygems.show"
```
Comment (arthurnn) [inline test/unit/helpers/rubygems_helper_test.rb]: why was this needed?

Comment (simi) [inline test/unit/helpers/rubygems_helper_test.rb]: I think that's because of [view template key lookup](https://github.com/rails/rails/blob/0fb2d1a0ba3360e928a7ac9d20378d7f5fbf915f/actionview/lib/action_view/helpers/translation_helper.rb#L116) is used in helpers..

```ruby
link_to t(".links.rss") #=> (looks for @virtual_path + key.to_s)
```

I think helper should not depend on `@virtual_path`. It will be better to use full key path for those string in those helpers and move them out of "view template" keys.

File: config/locales/fr.yml (hunk 1 above)
Comment (olivierlacan) [inline config/locales/fr.yml]: Yeah, I guess. I just hate this style of semi-condescending verb-based actions in French. Adverbs always feel more proper and less cutesy.

File: test/unit/helpers/rubygems_helper_test.rb (hunk 2 above)
Comment (olivierlacan) [inline test/unit/helpers/rubygems_helper_test.rb]: Yeah I came to the same conclusion. This was just a stop-gap to get back to green.

**Comment by arthurnn** [comment]
thanks ❤️

**Comment by olivierlacan** [comment]
Nice! Thanks Arthur!
